apps/desktop-wallet/src/modals/SendModals/SendModal.tsx

```diff
@@ -157,15 +159,13 @@ function SendModal<PT extends { fromAddress: Address }, T extends PT>({
         if (error.includes('consolidating') || error.includes('consolidate')) {
           setIsConsolidateUTXOsModalVisible(true)
           setConsolidationRequired(true)
-          posthog.capture('Error', { message: 'Could not build tx, consolidation required' })
```

Comment:
I'll bring it back but not as an error but rather as an event that happened (consolidation is required)

Reply:
Sounds good.